src/app/util/ember-compatibility-functions.cpp

```diff
@@ -943,6 +943,12 @@ CHIP_ERROR WriteSingleClusterData(const SubjectDescriptor & aSubjectDescriptor,
         apWriteHandler->CacheACLCheckResult({ aPath, requestPrivilege });
     }
 
+    if (aPath.mDataVersion.HasValue() &&
```

Comment:
Per spec this check is _after_ the timed write check.  Just move it to right after that block, please.

Reply:
updated, thanks